src/App.css

```diff
@@ -1,5 +1,15 @@
 .App {
   text-align: center;
+  background-color: var(--background-color);
+}
+
+.modal-backdrop {
+  background-color: rgba(0, 0, 0, 0.447);
+  width: 100%;
+  height: 120vh;
+  position: fixed;
+  z-index: 3;
```

Comment:
Suggestion: You could use variables for z-index for making it more manageable at one place.